docs/components/modal.md

```diff
@@ -1,348 +1,752 @@
+/>
+```
+
+### maxHeight
+
+Maximum height of the content area. When content overflows, content area becomes scrollable. Default value is viewport height minus `200px` (`calc(100vh - 200px)`).
+
+<KButton @click="modal15Visible = true">Modal</KButton>
+<KModal
+  max-height="200px"
+  title="Modal"
+  :visible="modal15Visible"
+  @cancel="closeAllModals"
+  @proceed="closeAllModals"
+>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Euismod nisi porta lorem mollis aliquam ut porttitor leo. Felis donec et odio pellentesque. Consectetur adipiscing elit pellentesque habitant morbi tristique senectus et netus. In eu mi bibendum neque egestas. Aliquam eleifend mi in nulla posuere sollicitudin. Faucibus ornare suspendisse sed nisi lacus sed viverra. Elementum curabitur vitae nunc sed. Nulla porttitor massa id neque aliquam vestibulum morbi. At varius vel pharetra vel turpis nunc eget.</p>
+
+  <p>Diam sollicitudin tempor id eu nisl nunc. Nunc mi ipsum faucibus vitae aliquet nec ullamcorper sit. Ullamcorper eget nulla facilisi etiam dignissim diam. Ultrices neque ornare aenean euismod elementum nisi. Enim tortor at auctor urna nunc id cursus. Et netus et malesuada fames ac turpis egestas maecenas. Nulla porttitor massa id neque aliquam vestibulum morbi blandit cursus. Euismod lacinia at quis risus sed vulputate odio ut enim. Sociis natoque penatibus et magnis dis parturient montes nascetur ridiculus. Eu turpis egestas pretium aenean pharetra magna ac placerat vestibulum. Justo laoreet sit amet cursus. Adipiscing enim eu turpis egestas pretium aenean pharetra. Ullamcorper velit sed ullamcorper morbi. Eleifend quam adipiscing vitae proin sagittis nisl rhoncus. Magna sit amet purus gravida quis blandit turpis cursus.</p>
+</KModal>
+
+```html
+<KModal
+  max-height="200px"
+  title="Modal"
+  :visible="modalVisible"
+  @cancel="handleModalClose"
+  @proceed="handleModalProceed"
+>
+  <p>Lorem ipsum dolor sit amet...
 </KModal>
 ```
 
 ### tabbableOptions
-Options to be passed to [`focus-trap`](https://github.com/focus-trap/focus-trap), which is responsible for trapping focus inside the modal box. If you're experiencing issues with testing `<KModal>` in `jsdom`, you can pass this prop according to the [focus-trap documentation](https://github.com/focus-trap/focus-trap#testing-in-jsdom).
+
+Options to be passed to [`focus-trap`](https://github.com/focus-trap/focus-trap), which is responsible for trapping focus inside the modal box.
 
 ## Slots
 
-There are 4 designated slots you can use to display content in the modal.
+### default
 
-- `header-image` - content displayed above the header, ignores padding
-- `header-content`
-- `body-content`
-- `footer-content` - Contains cancel & action buttons by default.
-- `action-buttons` - Contains action buttons which are right-aligned. This slot will not exist if using `footer-content` slot.
+Slot for modal content.
```

Comment:
Can you mention something here about the differences between `default` and `content` slots?